File: src/main/java/com/raiderrobotix/_2020/commands/Teleop.java
package com.raiderrobotix._2020.commands;

import static com.raiderrobotix._2020.OperatorInterface.input;
import static com.raiderrobotix._2020.subsystems.DriveBase.drives;

public final class Teleop extends Command {

	public Teleop() {
		requires(drives);
	}

	@Override
	protected final void execute() {
		drives.setSpeed(-input.getLeftY(), -input.getRightY());
	}

	@Override
	protected boolean isFinished() {
		return false;
	}

	@Override
	protected void initialize() {
	}

	@Override
	protected void end() {
		drives.setSpeed(0);
	}

}

File: src/main/kotlin/com/raiderrobotix/_2020/commands/Teleop.kt
package com.raiderrobotix._2020.commands

import com.raiderrobotix._2020.OperatorInterface
import com.raiderrobotix._2020.subsystems.DriveBase
import kotlinx.coroutines.GlobalScope
import org.team2471.frc.lib.coroutines.meanlibLaunch
import org.team2471.frc.lib.coroutines.periodic
import org.team2471.frc.lib.framework.use

suspend fun operatorControl() = GlobalScope.meanlibLaunch {
	use(DriveBase) {
		periodic {
			DriveBase.tankDrive(
				leftSpeed = -OperatorInterface.leftY,
				rightSpeed = -OperatorInterface.rightY
			)
		}
	}
}
